package.json

```diff
@@ -67,8 +68,8 @@
     "prettier": "^3.2.5",
     "supabase": "1.167.4",
     "ts-jest": "29.1.2",
-    "typescript": "^5.3.3",
-    "wrangler": "3.57.0"
+    "typescript": "5.6.2",
```

Comment:
I noticed this version of TypeScript throws a warning with our linter but I suppose we can wait for the linter to upgrade too.

Reply:
Thank you for noticing. I need to update it because `--noCheck` is a flag that was introduced in late versions of TypeScript with `tsc`.